freqtrade/rpc/api_server/webserver.py

```diff
@@ -101,7 +130,10 @@ def shutdown(cls):
         cls._rpc = None
 
     def send_msg(self, msg: Dict[str, str]) -> None:
```

Comment:
i'm currently spammed with `freqtrade.rpc.rpc_manager - ERROR - Message type 'whitelist' not implemented by handler telegram.`.

as that's a known "won't implement" handler for telegram, it would be great if we could add that to a "dummy-if" in it's `send_msg()` implementation (basically ignoring messages we know telegram won't be handling)

Reply:
it's an option - the main reason for the error was to detect if we "forget" to implement a messagetype for whatever reason.

I'm not sure how likely that is - but especially now while adding additional types it's a certain risk - where i'd rather take explicit decisions on "saw this - ignored this" - than "ups, forgot to implement".

but yeah - you can also omit the error (i suspect that'll break a test though).

Reply:
I've added a try-except block in `send_msg` to simply skip if it catches NotImplementedError